### DISCUSSION TITLE
ch4-rpc/ch4-02-pb-intro

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#636] [General]
# ch4-rpc/ch4-02-pb-intro

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch4-rpc/ch4-02-pb-intro.html

**Comment by pizicaiman** [comment]
4.2.1 hello.proto的代码需加一行
option go_package="./;hello";

option go_package = "path;name";
 
path 表示生成的go文件的存放地址，会自动生成目录的。
name 表示生成的go文件所属的包名

**Reply by meowrain** [reply]
谢谢😊

**Comment by Laurus-0121** [comment]
4.2.2 生成可执行文件在显示
WARNING: Package "github.com/golang/protobuf/protoc-gen-go/generator" is deprecated.
        A future release of golang/protobuf will delete this package,
        which has long been excluded from the compatibility promise.
之后就生成不了了？

**Reply by meowrain** [reply]
那就`go get google.golang.org/protobuf `
